parts/linux/cloud-init/artifacts/components.json

```diff
@@ -400,6 +400,11 @@
             "versions": [
               "1.1.12-ubuntu22.04u1"
             ]
+          },
+          "r2404": {
+            "versions": [
+              "1.1.12-ubuntu24.04u1"
```

Comment:
did we test to make sure this exists in PMC?

Reply:
Yes I checked, it exists :)